cordapp-example/contracts-kotlin/src/main/kotlin/com/example/schema/CashSchema.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.schema
+
+import net.corda.core.contracts.Amount
+import net.corda.core.schemas.MappedSchema
+import net.corda.core.schemas.PersistentState
+import java.util.*
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Table
+
+object CashSchema
+
+object CashSchemaV1 : MappedSchema(
+        schemaFamily = CashSchema.javaClass,
+        version = 1,
+        mappedTypes = listOf(PersistentCash::class.java)) {
+    const val CURRENCY = "PLN"
+
+    @Entity
```

Comment:
Name the table same as your state name, why did you name it `coin` if it's `Cash` state and `CashSchema`? Please rename to `cash_states`.